rollup.config.js

```diff
@@ -73,6 +73,15 @@ const makePlugins = isProduction =>
           sourceMap: true,
         },
       },
+      tsconfigOverride: {
+        compilerOptions: {
+          declaration: !isProduction,
+          declarationDir: './dist/types',
+          target: 'es6',
```

Comment:
I don't know if my intuition is correct, but do we wanna target `es5` for more browser/runtime support?

Reply:
Good eye, that gets taken care of by the `babel` step below 😉 So basically we have Rollup first run the TS plugin, which will typecheck the source and compile to ES6, and then `rollup-plugin-babel` comes in and transpiles the ES6 down to ES5. So a two-step process. We do this on `urql` and `renature` as well. If we were real hard core we'd also run it through `buble`, but I've had some tricky experiences getting `buble` to handle `async / await`. Typically you want to transform your `async / await` code to `Promises` first so you never have to bring in `regenerator-runtime` or any of that nonsense.

Reply:
Ohh gotcha, cool! `bublé` looks sweet -- what benefits could we get from it in this case? It seems like a smol, fast alternative to `babel`,  but do these things matter all that much for build tools anyway?

That and it may have [extra support for IE11](Unless you need to support IE11, you probably don't need to use Bublé to convert your code to ES5.):
> Unless you need to support IE11, you probably don't need to use Bublé to convert your code to ES5.

Reply:
I do think it'd be great to get IE11 support! Also, if I recall a conversation with Phil correctly, I believe it does a better job handling the spread operator and passing in `Object.assign`. I think we can use `buble` to do a strong first pass, and then hand off to Babel to transform `async` / `await` to `Promises`.